scipy/stats/tests/test_distributions.py

```diff
@@ -3990,16 +3990,17 @@ def test_ppf(self):
     # def chisq_entropy_mpmath(df):
     #     df = mp.mpf(df)
     #     half_df = 0.5 * df
-    #     entropy = half_df + mp.log(2) + mp.log(mp.gamma(half_df)) + \
-    #              (1 - half_df) * mp.digamma(half_df)
+    #     entropy = (half_df + mp.log(2) + mp.log(mp.gamma(half_df)) +
+    #                (mp.one - half_df) * mp.digamma(half_df))
     # return float(entropy)
 
     @pytest.mark.parametrize('df, ref',
-                             [(1, 0.7837571104739337),
+                             [(1e-4, -19988.980448690163),
+                              (1, 0.7837571104739337),
                               (100, 4.061397128938114),
-                              (10000, 6.370615639472648)])
+                              (1e15, 19.034900320939986)])
```

Comment:
Let's include a test that is just past the threshold for switching to the asymptotic formula.  This would have caught the `2/half_df` typo.
```suggestion
                              (251, 4.525577254045129),
                              (1e15, 19.034900320939986)])
```